app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -19,31 +21,51 @@
 import org.mozilla.vrbrowser.R;
 import org.mozilla.vrbrowser.SettingsStore;
 import org.mozilla.vrbrowser.search.SearchEngine;
+import org.mozilla.vrbrowser.utils.UrlUtils;
+
+import java.net.URI;
+import java.util.HashSet;
+import static java.lang.Math.toIntExact;
 
 
 public class TelemetryWrapper {
     private final static String APP_NAME = "FirefoxReality";
+    private final static String LOGTAG = "VRB";
+    private final static int HISTOGRAM_SIZE = 200;
+    private final static int BUCKET_SIZE_MS = 100;
+    private final static int HISTOGRAM_MIN_INDEX = 0;
```

Comment:
I move this line above for consistency/alphabetical order

Reply:
done